### ISSUE TITLE
Causing expensive memory costs by using JSON.parse in 2.5.0

### ISSUE THREAD

**Issue by wendycan** [getsentry/raven-node#451]
When I upgraded `raven-node` package from `2.4.2` to `2.5.0`, I find it will cause expensive memory costs. Sometimes my nodeJS project memory will reach to 400mb, but it used to be 100mb. On the other hand, it will spend more respond time(up to 2s). After removing `raven` error handler, my api will respond in 50ms.

```js
// in my project
Raven.config(SENTRY_DSN).install();
app.use(Raven.requestHandler());

app.use(Raven.errorHandler());  // !!!Remove it!!!

app.use((err, req, res, next) => {
       res.json(500, err.toString());
       next();
});
```

Checking the `raven-node` commits since 2.4.2, I found the below changes:

```js
-      kwargs = kwargs || {};
+      // Do not use reference, as we'll modify this object later on
+      kwargs = kwargs ? JSON.parse(stringify(kwargs)) : {};
```
[[commit] fix: Do not override context when capturing non-error exceptions](https://github.com/getsentry/raven-node/commit/b5e306c41d1425843bc12eb22d8e7d825b113ddf)

After I discarded above changes, the memory returned to normal and get a faster respond. 

`JSON.parse()` on a large object will spend much more memory. Hope to fix this problem in the new version~

**Comment by mitsuhiko** [comment]
Agreed. This should be a shallow copy instead.

**Comment by pawelniewie** [comment]
Had the same issue, reverting to 2.4.2 fixed it.

**Comment by kamilogorek** [comment]
@wendycan @pawelniewie fixed in `2.6.0`. Sorry for the inconvenience.
